tests/mapdata/test_mapdata.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import geopandas
+import shapely
+from map2loop.mapdata import MapData 
+from map2loop.m2l_enums import Datatype
+
+def test_structures_less_than_360():
```

Comment:
Isn't this testing greater than 360?

Reply:
changed to verbose test name in https://github.com/Loop3D/map2loop/pull/91/commits/6ffe6bf31ab860ee054bb175a63c00fee7b8a7ff to avoid confusion. 
The line `assert md.data[Datatype.STRUCTURE]['DIPDIR'].all() < 360` tests if all values are below 360, and if not, the test fails. Usually the convention is to name the test with the expected result of the assertion.